lib/options.js

```diff
@@ -105,7 +109,7 @@ module.exports = {
         description: 'Specific properties (parts of a request/response pair) to ignore during validation.' +
           ' Must be sent as an array of strings. Valid inputs in the array: PATHVARIABLE, QUERYPARAM,' +
           ' HEADER, BODY, RESPONSE_HEADER, RESPONSE_BODY',
-        external: true
+        external: false
```

Comment:
this could be a problem
we don't want these to show up in the importer options, but they should be allowed to be set from outside during validatation

Reply:
Should we remove this property and add a `type` property? As in type = conversion and type = validation?

Reply:
yep
ideally, options should be passed to each function of the schemaPack separately
adding a type: 'convert' / type: 'validate' would aid in that effort

Reply:
and what about `schemaFaker` option. Let's say we remove the external property completely, and add a type property with the value being one of `convert` and `validate`. `schemaFaker` would have value as `convert` and so it would show up in the importer options UI in App